### DISCUSSION TITLE
how to decompress tar file?

### DISCUSSION THREAD

**Discussion by yangyu2010** [tsolomko/SWCompression#12] [Q&A] [answered]

**Comment by tsolomko** [comment, marked as answer]
You may use the [TarContainer.open(container:)](https://tsolomko.github.io/SWCompression/Classes/TarContainer.html#/s:13SWCompression12TarContainerC4open9containerSayAA0B5EntryVG10Foundation4DataV_tKFZ) function, where the `container` argument is your tar file.
